src/Hazelcast.Net/Protocol/Codecs/SqlCloseCodec.cs

```diff
@@ -0,0 +1,116 @@
+// Copyright (c) 2008-2021, Hazelcast, Inc. All Rights Reserved.
```

Comment:
For this codec, and all other codecs and custom codecs: I assume you generated them via the protocol repository (eg the `hz generate-codecs` command)? So you probably made some changes to the protocol repo? Can you push a PR for that repo, and update the submodule accordingly?
(ping me for infos)

Reply:
Already have, forgot to mention: https://github.com/hazelcast/hazelcast-client-protocol/pull/382

Reply:
Merged and updated submodule ref